Distance_3/include/CGAL/squared_distance_3_2.h

```diff
@@ -389,6 +390,27 @@ squared_distance(const Triangle_3<K> & t,
   return internal::squared_distance(pt, t, K());
 }
 
+template <class K>
+inline
+typename K::FT
+squared_distance(const Tetrahedron_3<K> & t,
+                 const Point_3<K> & pt) {
+  if (t.has_on_boundary(pt) || t.has_on_bounded_side(pt))
```

Comment:
`has_on_unbounded_side`?

Reply:
That is the interior of the tetrahedron, right?

Reply:
What Marc suggests is to replace the two tests by one  (and a negation).